### ISSUE TITLE
PRD: JetBrains Cody Free --> Pro upsells

### ISSUE THREAD

**Issue by chenkc805** [sourcegraph/jetbrains#167]
### Description:

re: this thread -https://sourcegraph.slack.com/archives/C066F85TE80/p1701806871448459
I was able to get the rate limit to work, but:
1. Since this is a Free --> Pro tier limit that the user is hitting, they should see an "Upsell" message
  - Let's use this copy:
    - You've used up your chat and commands for the month: You've used all {LIMIT} chat messages and commands for the month. Upgrade to Cody Pro for unlimited autocompletes, chats, and commands. Upgrade[URL: /cody/subscription] or learn more[URL: /cody/subscription].
      - Yes I know the links are the same, this is intentional :slightly_smiling_face:
  - If Pro users hit this limit for abuse reasons, it's ok to show the copy we have live today for these userss.
2. "Upgrade" or "Learn more" buttons in JetBrains?
  - I see we can use links, but buttons would be clearer here.
  - Are we planning to ship this for Dec GA?
3. Also I don't see the "Subscriptions" tab
  - When's this slated to land?
4. What currently happens when the user hits their autocomplete limit in JetBrains?

### Design Tasks:

```[tasklist]
### Design Tasks
```

### Engineering Tasks:

```[tasklist]
### Engineering Tasks
- [ ] https://github.com/sourcegraph/jetbrains/pull/183
```

### Related Docs:

- [PRD: Your-Title]()

**Comment by taylorsperry** [comment]
Hey, @chenkc805, I'm not sure I follow this issue. Can you help me understand what work still needs to be done here?

**Comment by chenkc805** [comment]
This is super out of date. I think once we land https://github.com/sourcegraph/jetbrains/pull/183 we can close this one. If I find more bugs during our QA session going into February launch I"ll create a new issue.

**Comment by Sa1to** [comment]
@taylorsperry There is an issue with different values of rate limits that we can from agent api and one displayed on the website @vdavid created PR to hide limit values all together https://github.com/sourcegraph/jetbrains/pull/274 How should we handle this task in context of the issue that i mentioned?

**Comment by taylorsperry** [comment]
Chats/commands: 
"You've used all your chat messages and commands for the month. [Upgrade to Pro](https://sourcegraph.com/cody/subscription) for unlimited usage."

Autocompletions: 
"You've used all your autocompletions for the month. [Upgrade to Pro](https://sourcegraph.com/cody/subscription) for unlimited usage."
